streamlineTools/content.js

```
@@ -1,16 +1,28 @@
+    // commentHeader = "";
+  // }
+// } catch (e) {
+  // console.log(e);
+// }
 // let commentHeader = ''
-let code = "";
-let testCode = "";
+// let code = "";
+// let testCode = "";
+
+if(typeof commentHeader === "undefined"){
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)